src/Sulu/Bundle/SecurityBundle/DependencyInjection/Configuration.php

```diff
@@ -44,6 +45,40 @@ public function getConfigTreeBuilder()
                 ->end()
             ->end();
 
+
+        $this->addObjectsSection($rootNode);
+
         return $treeBuilder;
     }
+
+    /**
+     * Adds `objects` section.
+     *
+     * @param ArrayNodeDefinition $node
+     */
+    private function addObjectsSection(ArrayNodeDefinition $node)
```

Comment:
i have created a issue for that maybe we can discuss this there: https://github.com/sulu-io/sulu/issues/1311